src/tools/base/BaseSegmentationTool.js

```diff
@@ -0,0 +1,94 @@
+import BaseTool from './BaseTool.js';
```

Comment:
On the chopping block

![https://media0.giphy.com/media/12gxeCI1BGKAj6/giphy.gif?cid=790b7611ba0f6d1acbf8d345e6039f02956cc33747091fe5&rid=giphy.gif](https://media0.giphy.com/media/12gxeCI1BGKAj6/giphy.gif?cid=790b7611ba0f6d1acbf8d345e6039f02956cc33747091fe5&rid=giphy.gif)